client/components/status/status.js

```
@@ -1250,61 +1355,144 @@ export default class Status extends React.Component {
-                            </p>
-                          </div>
-                        ),
-                    )}
-                  {warningMessage && (
-                    <p className="important">
-                      <strong>{gettext(warningMessage)}</strong>
+              <div className="usage-overview bg row">
+                <div className="usage-overview-title">
+                  Daily usage overview
+                </div>
+                <p>Track how much time and data you've used today</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)